exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/Metadata.java

```diff
@@ -163,7 +163,9 @@ private ParquetTableMetadata_v1 getParquetTableMetadata(String path) throws IOEx
     watch.start();
     List<FileStatus> fileStatuses = getFileStatuses(fileStatus);
     logger.info("Took {} ms to get file statuses", watch.elapsed(TimeUnit.MILLISECONDS));
-    return getParquetTableMetadata(fileStatuses);
+    ParquetTableMetadata_v1 metadata_v1 = getParquetTableMetadata(fileStatuses);
+    logger.info("Took {} ms to read file metadata", watch.elapsed(TimeUnit.MILLISECONDS));
```

Comment:
Is this time supposed to be only for reading file metadata ? This is reporting the time inclusive of getFileStatuses().